Header header

# Coordinate System types
uint8 BEAM = 0
uint8 INSTRUMENT = 1
uint8 SHIP = 2
uint8 EARTH = 3

uint8 coordinate_system

bool use_attitude

bool use_3beam_solution

bool use_bin_mapping
